### PR TITLE
Add Reply message type

### PR DESCRIPTION
**Pull request by marcoconti83**

### CODE REVIEW

File: proto/messages.proto [hunk 1]
```diff
@@ -215,7 +225,6 @@ message Asset {
 //  optional Preview preview = 2;  // deprecated - preview was completely replaced
   oneof status {
     NotUploaded not_uploaded = 3;
-    RemoteData uploaded = 4;
```
Comment (mikeger) [inline proto/messages.proto]: Not sure about the reason why this must go now.

Comment (gregor) [reply proto/messages.proto]: Definitely removed by accident

Comment (marcoconti83) [reply proto/messages.proto]: Yep

Comment (marcoconti83) [reply proto/messages.proto]: Fixed

**Comment by marcoconti83** [comment]
Sold!

**Pull request merged**